youtube_dl/extractor/vvvvid.py

```diff
@@ -156,3 +156,39 @@ def f(m):
             'view_count': int_or_none(video_data.get('views')),
             'like_count': int_or_none(video_data.get('video_likes')),
         }
+
+
+class VVVVIDPlaylistIE(VVVVIDIE):
+    _VALID_URL = r'https?://(?:www\.)?vvvvid\.it/(?:#!)?(?:show)/(?P<show_id>\d+)/(?P<id>[^/]+)$'
```

Comment:
should match urls with query, fragment, etc...(https://www.vvvvid.it/show/156/psycho-pass?foo=bar, https://www.vvvvid.it/show/156/psycho-pass/, ...).

Reply:
improved regex in https://github.com/ytdl-org/youtube-dl/pull/27574/commits/c73c6ec37fd103a8918c702e111ebfb1ab75b9d4